src/neuroconv/datainterfaces/ecephys/phy/phydatainterface.py

```diff
@@ -23,6 +27,23 @@ def get_source_schema(cls) -> dict:
         ] = "Path to the output Phy folder (containing the params.py)."
         return source_schema
 
+    def get_max_channel(self):
+        folder_path = Path(self.source_data["folder_path"])
+
+        templates = np.load(str(folder_path / "templates.npy"))
+        channel_map = np.load(str(folder_path / "channel_map.npy")).T
+        whitening_mat_inv = np.load(str(folder_path / "whitening_mat_inv.npy"))
+        templates_unwh = templates @ whitening_mat_inv
+
+        cluster_ids = self.sorting_extractor.get_property("original_cluster_id")
+        templates = templates_unwh[cluster_ids]
+
+        max_over_time = np.max(templates, axis=1)
+        idx_max_channel = np.argmax(max_over_time, axis=1)
+        max_channel = channel_map[idx_max_channel].ravel()
+
+        return max_channel
```

Comment:
Also, two other questions here:
1) Aren't the templates scaled? I am thinking on the `amplitudes.npy` 
2) Also, is `max_channel` meanigful if spikes are negative? I think here it is consistent but aren't most people interested on the absolute value largest. On the templates data base we use "best channel" which did not have that load.

Now I am aware that you are using machinery that is already there in the `add_units_table` but I wanted to point this out.

Reply:
1. Scaled how? In a way that would affect which channel is selected?
2. yeah, good point, the argmax(abs()) might be better

Reply:
Yeah, I shared this with a friend that uses Phy and he flagged that. Reading the documentation I feel less certain.

We could do a roun-trip with spikeinterface artificial data and the export to phy functionality to see if your function gets it right.

Reply:
There is some code by Nick Stteinmetz that calculates templates max without using the amplitudes (but they don't do the de-whitening step that you do)

https://github.com/cortex-lab/spikes/blob/master/analysis/templatePositionsAmplitudes.m

Reply:
I think he does. Look at winv

Reply:
But the block where the max channel is calculated take the temps as they come directly from the input:

https://github.com/cortex-lab/spikes/blob/fcea2b20e736b533e5baf612752b66121a691128/analysis/templatePositionsAmplitudes.m#L64-L71

Am I missing something?